frontend/src/components/SocialMediaButton/index.js

```diff
@@ -1,5 +1,6 @@
 import "./index.css";
 
+import Default from "../../assets/imgs/socialButtons/default.png";
```

Comment:
Why do you need this?

Reply:
The idea was if the prop name was empty, it should show a default image. Should I remove it?

Reply:
Can the `name` be empty?